Journals_API.ipynb

```diff
@@ -0,0 +1,506 @@
+            "                    \"OTHER\": 1\n",
+            "                }\n",
+            "            },\n",
+            "            {\n",
+            "                \"rows-omitted\": \"[...]\",\n",
+            "            },\n",
+            "            {\n",
+            "                \"volume\": \"57\",\n",
+            "                \"year\": \"2019\",\n",
+            "                \"refsources\": {\n",
+            "                    \"CROSSREF\": 15,\n",
+            "                    \"AUTHOR\": 1\n",
+            "                }\n",
+            "            },\n",
+            "            {\n",
+            "                \"volume\": \"58\",\n",
+            "                \"year\": \"2020\",\n",
+            "                \"refsources\": {\n",
+            "                    \"CROSSREF\": 16\n",
+            "                }\n",
+            "            },\n",
+            "            {\n",
+            "                \"volume\": \"59\",\n",
+            "                \"year\": \"2021\",\n",
+            "                \"refsources\": {\n",
+            "                    \"PUBLISHER\": 11,\n",
+            "                    \"AUTHOR\": 2\n",
+            "                }\n",
+            "            }\n",
+            "        ]\n",
+            "    }\n",
+            "}\n"
+          ]
+        },
+        {
+          "name": "stderr",
+          "output_type": "stream",
+          "text": [
+            "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
+            "                                 Dload  Upload   Total   Spent    Left  Speed\n",
+            "100  5119  100  5119    0     0  12465      0 --:--:-- --:--:-- --:--:-- 12861\n"
+          ]
+        }
+      ],
+      "source": [
+        "%%bash\n",
+        "token=\"your-token-here\"\n",
+        "# note: add | python -m json.tool to the end of the request to print the response in a more readable format\n",
+        "curl -H \"Authorization: Bearer $token\" https:\/\/api.adsabs.harvard.edu\/v1\/journals\/Refsource\/ARA&A' | python3 -m json.tool"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "### `/Holdings`\n",
+        "Generate a list of available electronic fulltext sources for a given `<bibstem>/<volume>`\n",
+        "\n",
+        "*Return data*: output is JSON\n",
+        "* `bibstem` (text): ADS bibstem for a given Title\n",
+        "* `volume` (text): ADS bibstem for a given Title\n",
+        "* `numFound` (integer): Number of records found for Title/volume\n",
+        "* `holdings` (list, dict): page number and e-sources for each indexed paper in the volume\n",
+        "  * `page` (text): page number or page id\n",
+        "  * `esources` (list, text): one or more of [`EPRINT_HTML`, `EPRINT_PDF`, `PUB_HTML`, `PUB_PDF`, 'ADS_PDF`, `ADS_SCAN`, `AUTHOR_PDF`, `AUTHOR_HTML`]\n"
```

Comment:
Typo - the tick in front of ADS_PDF should be a backtick - the formatting gets all weird when rendered

Reply:
Fixed